.eslintrc

```diff
@@ -37,6 +37,7 @@
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/jsx-one-expression-per-line": "off",
     "react/jsx-props-no-spreading": "off",
+    "react/static-property-placement": ["warn", "property assignment"],
```

Comment:
For
`  14:5  error  'propTypes' should be declared outside the class body  react/static-property-placement`